Formula/postgis.rb

```diff
@@ -52,13 +52,13 @@ def install
     mkdir "stage"
     system "make", "install", "DESTDIR=#{buildpath}/stage"
 
-    bin.install Dir["stage/**/bin/*"]
-    lib.install Dir["stage/**/lib/*"]
+    bin.install Dir["stage/#{HOMEBREW_PREFIX}/**/bin/*"]
+    lib.install Dir["stage/**/lib/*", "stage/#{HOMEBREW_PREFIX}/**/lib/*"]
     include.install Dir["stage/**/include/*"]
```

Comment:
There are `lib` files under _/tmp/.../stage/usr/local/..._ and under _/tmp/../stage/home/linux/.linuxbrew/..._. The original `Dir[]` is meant to match on both but due to the dotfile issue it would ignore the files under _.linuxbrew_ so a second pattern makes sure that those files are also included.

`include` (C header) files only seem to exist under _.../stage/usr/local/..._ so adding the `#{HOMEBREW_PREFIX}` is not needed (it would actually break the formula).

These are the only two special cases, everything else under `stage` is in `#{HOMEBREW_PREFIX}`.

While this solution works and does not require a custom `dir()` method with special globing, I think it's a little more error prone: what if, for example, if new executables show up under _.../stage/usr/local/bin/..._ in some future version of PostGIS? The formula as is would silently ignore those (because with this change it only looks for files under the `#{HOMEBREW_PREFIX}` directory.